exec/local.go

```diff
@@ -154,6 +154,7 @@ func bufferOutput(ctx context.Context, task *Task, out sliceio.Reader) (buf task
 			err = errors.E(err, errors.Fatal)
 		}
 	}()
+	shards := make([]int, *defaultChunksize)
```

Comment:
minor point, but this seems a little dangerous to be indirecting here, from what I can see defaultChunksize is initialized from another variable and consequently on the initialization order of those variables. We got bitten by this in go 1.13 since the order of such initializations is not defined across files.

Reply:
I don't think there's a problem here: this code is called only after static initialization, so all vars are fully initialized regardless of order..